readthedocs/templates/projects/project_delete.html

```diff
@@ -6,6 +6,11 @@
 {% block content-header %}<h1>{% blocktrans with project.name as name %}Delete {{ name }}?{% endblocktrans %}</h1>{% endblock %}
 
 {% block content %}
+
+  {% if is_superproject %}
+    <p>This project has subprojects under it. Deleting this will make them as regular projects. This will effect the URLs of the subprojects and they will be served normally as other projects.</p>
```

Comment:
We should put a link to the subprojects page here `has subprojects...`

Also, the paragraph needs to be marked for translation